content/en/docs/tasks/security/cert-management/plugin-ca-cert/index.md

```diff
@@ -4,7 +4,7 @@ description: Shows how system administrators can configure Istio's CA with an ex
 weight: 80
 keywords: [security,certificates]
 aliases:
-    - /docs/tasks/security/plugin-ca-cert/
+    - /docs/tasks/security/cert-management/plugin-ca-cert/
```

Comment:
This is wrong. An alias is the old location of a moved file.
```suggestion
    - /docs/tasks/security/plugin-ca-cert/
```

Reply:
Also note that you need to add an alias to the other doc you moved. Add the following a the top of `/docs/tasks/security/cert-management/dns-cert/index.md`.
```
aliases:
    - /docs/tasks/security/dns-cert/
```

Reply:
Sure. I'll do that.